README.MD

```diff
@@ -405,6 +407,45 @@ export class JsonApiMetaModel {
 }
 ```
 
+### Datastore config
+
+Datastore config can be specified through the `JsonApiDatastoreConfig` decorator and/or by setting a `config` variable of the `Datastore` class. If an option is specified in both objects, a value from `config` variable will be taken into account.
+
+```typescript
+@JsonApiDatastoreConfig(config: DatastoreConfig)
+export class Datastore extends JsonApiDatastore {
+    private customConfig: DatastoreConfig = {
+        baseUrl: 'http://something.com'
+    }
+
+    constructor() {
+        this.config = this.customConfig;
+    }
+}
+
+`config`:
```

Comment:
I guess you didn't want this to be included in the code section, so the end of code indicator needs to be here.